### PR TITLE
feat: app content

### PR DESCRIPTION
**Pull request by snack-0verflow**
# Description

- Dispatch redux action on app init
- Fetch featured blog content via `ContentService`
- Display featured content via `FeaturedContentList`

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [x] I have run `npm test`
- [x] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

**Review by selvach — APPROVED** [review]

**Comment by selvach** [comment]
@snack-0verflow Are the changes to es.json purely formatting?

**Review by petermason — APPROVED** [review]

File: src/components/Content/FeaturedContentList.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const home = useSelector<RootState, Optional<IFeaturedContent[]>>((state) => state.content.featuredHome);
+  const thankyou = useSelector<RootState, Optional<IFeaturedContent[]>>((state) => state.content.featuredThankyou);
+
+  const mapper = (item: IFeaturedContent) => (
+    <View testID="featured-content-callout" key={item.slug}>
+      <ExternalCallout
+        link={item.link}
+        calloutID={item.slug}
+        imageSource={{ uri: item.thumbnail_image_url }}
+        aspectRatio={1029 / 600}
```
Comment (the-kenneth) [inline src/components/Content/FeaturedContentList.tsx:30]: This should probably be height / width incase the image changes size in the future?

**Review by the-kenneth — CHANGES_REQUESTED** [review]

**Comment by snack-0verflow** [comment]
> @snack-0verflow Are the changes to es.json purely formatting?

Yes

File: src/components/Content/FeaturedContentList.tsx (hunk 1 above)
Comment (snack-0verflow) [reply src/components/Content/FeaturedContentList.tsx:30]: Atm is assuming the content size is the same, as the backend content doesn't have sizes specified

Comment (the-kenneth) [reply src/components/Content/FeaturedContentList.tsx:30]: Is it not possible to get the sizing dynamically? If its not easy we can leave it for now

Comment (selvach) [reply src/components/Content/FeaturedContentList.tsx]: The aspectRatio was something I introduced in order to get the correctly center the card in the parent container and have the image fill the space. There might be a better way to this which doesn't requite the aspect ratio....

**Review by the-kenneth — APPROVED** [review]

**Pull request merged**